tools/ci_build/github/android/build_aar_package.py

```diff
@@ -180,6 +180,9 @@ def _build_aar(args):
         "-DminSdkVer=" + str(build_settings["android_min_sdk_version"]),
         "-DtargetSdkVer=" + str(build_settings["android_target_sdk_version"]),
         "-DbuildVariant=" + str(build_settings["build_variant"]),
+        "-DENABLE_TRAINING_APIS=1"
+        if "--enable_training_apis" in build_settings["build_params"]
+        else "-DENABLE_TRAINING_APIS=0",
```

Comment:
```suggestion
        "-DENABLE_TRAINING_APIS=1" if "--enable_training_apis" in build_settings["build_params"]
                                   else "-DENABLE_TRAINING_APIS=0",
```

nit: hard to skim the names

Reply:
Will defer this to a different PR. Needs changes in lot of files.